LayoutTests/imported/w3c/resources/import-expectations.json

```diff
@@ -123,6 +123,7 @@
     "web-platform-tests/css/css-grid/layout-algorithm/": "import",
     "web-platform-tests/css/css-grid/parsing/": "import",
     "web-platform-tests/css/css-images": "import",
+    "web-platform-tests/css/css-images/image-set": "import",
```

Comment:
can you undo this change? it's covered by the previous line.

Reply:
Yes. Might be a nice-to-have FIXME which can detect if we're only updating a subdirectory.